src/libraries/System.Drawing.Common/Directory.Build.props

```diff
@@ -4,5 +4,8 @@
     <StrongNameKeyId>Open</StrongNameKeyId>
     <IncludePlatformAttributes>true</IncludePlatformAttributes>
     <UnsupportedOSPlatforms>browser</UnsupportedOSPlatforms>
+    <PackageVersion>5.0.1</PackageVersion>
+    <AssemblyVersion>5.0.0.1</AssemblyVersion>
+    <HarvestVersion>4.5.2</HarvestVersion>
```

Comment:
We should remove HarvestVersion now. It was only a temporary workaround to not have to wait for a 3.1 build.